src/blocks/social-profiles/edit.js

```diff
@@ -0,0 +1,493 @@
+											<div className="editor-url-input block-editor-url-input editor-url-input--coblocks">
+												<TextControl
+													value={ pinterest }
+													/* eslint-disable jsx-a11y/no-autofocus */
+													// Disable Reason: The rule is meant to prevent enabling auto-focus, not disabling it.
+													placeholder={ __( 'https://pinterest.com/' ) }
+													/* eslint-enable jsx-a11y/no-autofocus */
+													onChange={ value =>
+														setAttributes( { pinterest: value } )
+													}
+												/>
+											</div>
+										</form>
+									</Popover>
+								) }
+							</li>
+						) }
+						{ ( placeholder || ( linkedin || isSelected ) ) && (
+							<li>
+								<span
+									className={ classnames(
+										buttonClasses,
+										'wp-block-coblocks-social__button--linkedin',
+										{
+											'is-empty': ! linkedin,
+											'is-selected': this.state.currentIcon === 'linkedin',
+										}
+									) }
+									style={ buttonStyles }
+									onClick={ () => this.setState( { currentIcon: 'linkedin' } ) }
+								>
+									<span
+										className="wp-block-coblocks-social__icon"
+										style={ iconStyles }
+									/>
+									<span className="wp-block-coblocks-social__text">
+										{ __( 'Share on LinkedIn' ) }
+									</span>
+								</span>
+								{ this.state.currentIcon === 'linkedin' && (
+									<Popover>
+										<form
+											className="block-library-button__inline-link block-library-button__inline-link--coblocks"
+											onSubmit={ event => event.preventDefault() }
+										>
+											<Dashicon icon="admin-links" />
+											<div className="editor-url-input block-editor-url-input editor-url-input--coblocks">
+												<TextControl
+													value={ linkedin }
+													/* eslint-disable jsx-a11y/no-autofocus */
+													// Disable Reason: The rule is meant to prevent enabling auto-focus, not disabling it.
+													placeholder={ __( 'https://linkedin.com/' ) }
+													/* eslint-enable jsx-a11y/no-autofocus */
+													onChange={ value => setAttributes( { linkedin: value } ) }
+												/>
+											</div>
+										</form>
+									</Popover>
+								) }
+							</li>
+						) }
+						{ ( placeholder || ( youtube || isSelected ) ) && (
+							<li>
+								<span
+									className={ classnames(
+										buttonClasses,
+										'wp-block-coblocks-social__button--youtube',
+										{
+											'is-empty': ! youtube,
+											'is-selected': this.state.currentIcon === 'youtube',
+										}
+									) }
+									style={ buttonStyles }
+									onClick={ () => this.setState( { currentIcon: 'youtube' } ) }
+								>
+									<span
+										className="wp-block-coblocks-social__icon"
+										style={ iconStyles }
+									/>
+									<span className="wp-block-coblocks-social__text">
+										{ __( 'See us on Youtube' ) }
```

Comment:
Typo: Should be 'YouTube'

Reply:
And the "See us on" on this and other icons should be removed, and just print the icon label.